src/Uno.UI/DataBinding/BindingPath.BindingItem.cs

```diff
@@ -0,0 +1,524 @@
+﻿#nullable enable
```

Comment:
@Youssef1313  Is there any relevant change in this file that we should look at, or it's just a class extract?

Reply:
Just extract

Reply:
I can revert to ease review if necessary